### ISSUE TITLE
Calendar providers

### ISSUE THREAD

**Issue by adocampo** [SickRage/SickRage#3316]
I would like to developers to think about the possibiliy to develop a custom agenda for different countries.  So the user can select from the Global Agenda (TVDB or wherever we get the current information), where the information of the next episode aired is the original TV show date or their country's one, where the information is customized for the country she/he belongs to.
In Spain we have a web service: http://espoilertv.com/calendario/  I guess users from other countries can help with more providers, and perhaps add a new setting: calendar providers.

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by miigotu** [comment]
There is already a setting to convert it between local time and network time. It effects everything inclucing home, displayshow, and calendar.

**Comment by adocampo** [comment]
No, I think the option you say is mainly for USA, which have several time zones. If an episode is aired on the east coast and you live on the west coast, to know exactly when it will be aired on your location.

What I mean is, if a show originally aired on, for example, January 1st in USA and April 4th in Spain, show me that later date instead the first one. And on the other places: next episode, schedule... all places with the spanish agenda, not the original aired date. 

That is specially useful for those countries where movies and tv shows are dubbed or those non-english speakers who prefer to watch the series dubbed in their language.

Is just an idea, I know it is not easy to develop, as a reliable font for every country is needed, but it will be very useful for many.
